### ISSUE TITLE
LiveDVD asks for root password although it is empty

### ISSUE THREAD

**Issue by vinyanalista** [kamarada/kiwi-config-Kamarada#3]
When I use `su`, it asks for password and I need to just hit Enter:

```
linux@linux:~> su
Password: 
linux:/home/linux #
```

Same happens when I use `sudo`.

When I use `kdesu` (e.g. to open YaST in KDE), I need to just click on Ok or hit Enter.

When the screen gets locked, I need to press the space bar and click on Unlock.

When I log in as root on tty1, it asks for password and I need to just hit Enter:

```
linux login: root
Password:
Have a lot of fun...
linux:~ #
```

As the root password is empty, the login screen should not ask for it:

```
linux login: root
Have a lot of fun...
linux:~ #
```

This issue affects only openSUSE Leap 42.2-Alpha3:
- Release [42.2-Alpha3-20160824](https://github.com/kamarada/kiwi-config-Kamarada/releases/tag/42.2-Alpha3-20160824)

openSUSE Leap 42.1 is not affected by this issue:
- Release [42.1-20160806](https://github.com/kamarada/kiwi-config-Kamarada/releases/tag/42.1-20160806)

Any ideas on what is going wrong?

**Comment by vinyanalista** [comment]
[openSUSE Tumbleweed KDE Live DVD](https://en.opensuse.org/openSUSE:Tumbleweed_installation) does not ask for root password:

```
Welcome to openSUSE Tumbleweed 20160826 - Kernel 4.7.2-1-default (tty1).


new-host login: root
Have a lot of fun...
new-host:~ #
```

**Comment by vinyanalista** [comment]
Solved! I found that command in [the `config.sh` file for the openSUSE Tumbleweed LiveDVD](https://build.opensuse.org/package/view_file/openSUSE:Factory/kiwi-config-openSUSE/config.sh.in?expand=1):

``` sh
sed -i -e 's,^\(.*pam_gnome_keyring.so.*\),#\1,'  /etc/pam.d/common-auth-pc
```

It was the solution to the [openSUSE bug #544314](https://bugzilla.novell.com/show_bug.cgi?id=544314), `su` was requiring root password on a Live medium.

I added it in [my `config.sh`](https://github.com/kamarada/kiwi-config-Kamarada/blob/20734930fb88df613e5997ea29e7cdc58382200e/source/config.sh) file and the root password is not required anymore while on Live.

**Comment by vinyanalista** [comment]
For future reference, if one wants a passwordless Live medium of openSUSE Leap 42.2 (and maybe future versions too), here is what to do:
- `config.xml`:

``` xml
<users group='root'>
    <user name='root' pwd='' home='/root' shell='/bin/bash'/>
</users>
```
- `config.sh`:

``` sh
#======================================
# /etc/sudoers hack to fix #297695
# (Installation Live DVD: no need to ask for password of root)
# https://bugzilla.novell.com/show_bug.cgi?id=297695
#--------------------------------------
sed -i -e "s/ALL ALL=(ALL) ALL/ALL ALL=(ALL) NOPASSWD: ALL/" /etc/sudoers
chmod 0440 /etc/sudoers

/usr/sbin/useradd -m -u 999 linux -c "LiveDVD User" -p ""

# delete passwords
passwd -d root
passwd -d linux
# empty password is ok
pam-config -a --nullok

# bug 544314, we only want to disable the bit in common-auth-pc
# https://bugzilla.novell.com/show_bug.cgi?id=544314
sed -i -e 's,^\(.*pam_gnome_keyring.so.*\),#\1,'  /etc/pam.d/common-auth-pc
```

Source:
- https://github.com/kamarada/kiwi-config-Kamarada/commit/03751d46717a9b8e4d955e144160699ace1a0d80
- https://github.com/kamarada/kiwi-config-Kamarada/commit/20734930fb88df613e5997ea29e7cdc58382200e
